functions/src/helpers.ts

```diff
@@ -16,23 +16,63 @@ export const createMP3DBRef = (filePath, _module) => {
     const mp3 = parts[2];
     let file_name = mp3.slice(0, -4);
 
-    return `/${_module}/files/${list}/${file_name}`;
+    return `/files/${list}/${file_name}`;
 }
 
 export const extractListFromFilename = (fileName) => {
     return fileName.match(/^[^-]*[^ -]/g)[0];
 }
 
 
-export const storeFileNameToDB = async (filePath, db, _module) => {
+export const storeFileToDB = async (filePath, db, _module) => {
     if(checkValidMP3(filePath)) {
         let ref = db.ref(createMP3DBRef(filePath, _module));
 
         //checking if the file already exists in the RT db
         let snapshot = await ref.child("status").once('value')
 
         if(!snapshot.exists()) {
-            ref.set({status: "Spare"});
+            ref.set({
+                languages: ['test'],
+                languagesConfirmed: false,
+                notes: 'test',
+                soundQualityReporting: {
+                    status: 'Spare',
+                    followUp: 'test',
+                    timestampDone: 'test',
+                    timestampGiven: 'test',
+                    asignee: {
+                        name: 'test',
+                        emailAddress: 'test'
+                    }
+                },
+                contentReporting: {
+                    status: 'Spare',
+                    followUp: 'test',
+                    timestampDone: 'test',
+                    timestampGiven: 'test',
+                    asignee: {
+                        name: 'test',
+                        emailAddress: 'test'
+                    },
+                    soundQualitySubmission: {
+                        fileName: 'test',
+                        soundQualityRating: 'test',
+                        unwantedParts: 'test',
+                        soundIssues: 'test',
+                        duration: 'test',
+                        comments: 'test',
+                        author: {
+                            name: 'test',
+                            email_address: 'test'
+                        },
+                        token: 'test',
+                        created: 'test',
+                        completed: 'test',
+                        changed: 'test',
+                    }
+                }                
+            });
```

Comment:
Why this block has all the fields filled with “test”?

Reply:
This is what I meant when I talked about adding any **default value** [here](https://trello.com/c/rNQJaJUP/4-monitor-storage-and-add-file-entries-into-database#comment-5bed701a3c421272e679f8d4) just to make the data structure appear on the database.

Reply:
Oh, you said we don't need them right now ... my mistake

Reply:
Yes.